### DISCUSSION TITLE
.Net Core 3.1 support

### DISCUSSION THREAD

**Discussion by GuruDev-2021** [mkaring/ConfuserEx#364] [Q&A]
I am unable to run Confuser on my .Net Core 3.1 sample project, is it supported? if it is not supported, is there any plan to support it? 
If there is no plan to support, is there any alternatives for this?
My sample application is in .Net Core 3.1 and its a WPF Project.

**Comment by mkaring** [comment]
.NET Core and .NET 5 are not supported right now. Getting that to work is the top item on the to-do list.
